cleanlab/object_detection/filter.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Methods to find label issues in an object detection dataset (object detection data), where each annotated bounding box in an image receives its own class label."""
+
+from typing import List, Any, Dict
+import numpy as np
+
+from cleanlab.object_detection.rank import (
+    _get_valid_inputs_for_compute_scores,
+    _compute_overlooked_box_scores,
+    _compute_badloc_box_scores,
+    _compute_swap_box_scores,
+)
+
+
+def find_label_issues(
+    labels: List[Dict[str, Any]],
+    predictions: List[np.ndarray],
+) -> np.ndarray:
```

Comment:
consider an optional arg:
`frac_noise = 1.0` which has the same function as this arg in: `cleanlab.filter.find_label_issues()`

Reply:
make sure if you do add optional args though, you separate them from these two main args using: `, *, ` so that they are keyword only args like we have for the rest of library (allows us to re-order them in the future without breaking users' code)

Reply:
Good idea for follow-up PR!